test/test_util.h

```diff
@@ -1134,7 +1134,7 @@ std::ostream& operator<<(std::ostream& os, const CUB_NS_QUALIFIER::KeyValuePair<
 /**
  * Define for types
  */
-CUB_VEC_OVERLOAD(char, char)
+CUB_VEC_OVERLOAD(char, signed char)
```

Comment:
Are you missing the first `char`?
```suggestion
CUB_VEC_OVERLOAD(signed char, signed char)
```

Reply:
No, it's a name that is going to be turned into `char2`, `char4` by the macro.

Reply:
urgh, totally not tricky